packages/medusa/src/api/routes/admin/shipping-options/create-shipping-option.ts

```diff
@@ -203,15 +211,16 @@ export class AdminPostShippingOptionsReq {
   @IsString()
   provider_id: string
 
-  @IsOptional()
   @IsString()
-  profile_id?: string
+  profile_id: string
```

Comment:
todo: Looks like the profile_id is now required, but in the end point if it is not present we are retrieving the default one. So I would suggest the API to not required this property. wdyt?